### PR TITLE
Add python installer

### PR DESCRIPTION
**Pull request by ry**

### CODE REVIEW

**Comment by mirkojotic** [comment]
@ry 

Ok, Python makes sense (it's already a requirement ). No problem it was my pleasure!

Yeah if you're ok with it I will continue working on it.

File: tools/install.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+# Copyright 2018 the Deno authors. All rights reserved. MIT license.
+import os
+import json
+import sys
+import tempfile
+import shutil
+from urllib2 import urlopen
```
Comment (hayd) [inline tools/install.py]: This file can be python3 compatible:
```
from __future__ import print_function
try:
    from urllib.request import urlopen
except ImportError:
    from urllib2 import urlopen
```
and updating the print statements should suffice.

Comment (mirkojotic) [reply tools/install.py]: As soon as I get it working on all platforms I will make it p3 compatible @hayd

**Comment by ry** [comment]
@mirkojotic Yes please continue with it - all yours. Thank you!

Closing in favor of #639.

**Pull request closed without merging**